src/block_cache/block_chunkcache.c

```diff
@@ -184,75 +184,64 @@ __chunkcache_make_hash(WT_CHUNKCACHE *chunkcache, WT_CHUNKCACHE_HASHID *hash_id,
 }
 
 /*
- * __chunkcache_evict_one --
- *     Evict a single chunk from the chunk cache.
+ * __chunkcache_should_evict --
+ *     Decide if we can evict this chunk. In the current algorithm we only evict the chunks with the
+ *     zero access count. We always decrement the access count on the chunk that is given to us. The
+ *     thread accessing the chunk increments the access count. So we will only evict the chunks that
+ *     have not been accessed for a long time.
```

Comment:
`So we will only evict the chunks that have not been accessed for a long time.` Not exactly true - if a chunk is used just once, it will age out after one iteration of eviction.  Perhaps `We will only evict a chunk that has not be accessed for a time proportional to the number of accesses made to it.`